index.css

```diff
@@ -8,55 +8,105 @@
   }
 }
 
-@layer components {
-  .calendar {
-    @apply border-l-neutral-950
-  }
-  .border {
-    @apply border-black border-2
-  }
-  .btn {
-    @apply bg-green-700  py-3 px-6 font-serif text-sm;
-  }
-  .button {
-    @apply !text-primary-textColor !bg-primary-pageColor h-10 w-2 
-  }
-  .button:disabled {
-    @apply !bg-primary-blurColor !text-transparent
-  }
-  .button:enabled:hover,
-  .button:enabled:focus {
-    @apply !bg-primary-secondaryTabColorBorderColor !border-l-zinc-950
-  }
-  .react-calendar {
-    @apply  w-fit !border-l-zinc-950
-  }
-  .react-calendar__navigation__arrow {
-    @apply hidden
-  }
- 
+.react-calendar {
+  @apply bg-white;
+  border-spacing: 0;
+  border-collapse: collapse;
+  @apply rounded-none !important;
```

Comment:
@Harsh71019 , why we are using Important

Reply:
these are using it because we this default styles are coming from react calendar lib.